src/pages/hosts.js

```diff
@@ -3,28 +3,21 @@ import PropTypes from 'prop-types';
 
 import Hosts from '../components/Hosts';
 
-import withData from '../lib/withData';
 import withIntl from '../lib/withIntl';
-import withLoggedInUser from '../lib/withLoggedInUser';
+import { withUser } from '../components/UserProvider';
 
 class HostsPage extends React.Component {
   static propTypes = {
-    getLoggedInUser: PropTypes.func,
+    LoggedInUser: PropTypes.object,
   };
 
   constructor(props) {
     super(props);
     this.state = {};
```

Comment:
If the state is not used anymore we can remove it (as well as the `constructor`)

Reply:
Done.